client/model-config.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
why are some things remote and some in-memory?

I wish loopback would switch to an extended json format that allowed comments... @bajtos @raymondfeng yaml, json5, anything (while still supporting vanilla json)